### PR TITLE
Updated Commit

### PR DESCRIPTION
**Pull request by CodeByKarthik**
Hello Guys,

I've found the problem which causes error when installing the requirements of Carrot-Assistant. When we attempt to installing OpenSSL using pip, it throws an error. Also they were some other problems in conda level environment and the problem was in the Operating System level. 
 
So I created a script called (config.py) which will look for the type of OS and install all the package and dependencies and I've tested it in my local machine. The scripts are ready to install in other local machines. 

We need to make changes in the readme for lettuce repo. When Lettuce repo was cloned to the system, the user should run the following commands below:

-> chmod +x install_requirements.sh 

(This command will give permission to install the OpenSSH in the system. Since OpenSSH works in the OS level, the user needs to give permission for installing it). 

-> ./install_requirements.sh 

(This command will install all the packages and dependencies listed in "requirements.txt" and also checks whether OpenSSH is installed in the local computer. If OpenSSH is not available, the scripts will be installed and if OpenSSH version is not updated to the latest version, the scripts will update itself.).

Let me know for any changes and kindly do the needful.
Thanks and regards,
Karthik

### CODE REVIEW

File: Carrot-Assistant/config.py
Comment (kuraisle) [inline Carrot-Assistant/config.py:1]: As I said the other day, I don't think we should do this. OpenSS**L** is an important part of security over networks and we shouldn't be messing with users' OS security. In any case, this installs OpenSS**H**, which we should also not touch

File: install_requirements.sh
Comment (kuraisle) [inline install_requirements.sh:1]: As with the config.py, we shouldn't mess with users' security

**Review by kuraisle — CHANGES_REQUESTED** [review]
config.py should be removed, and install_requirements.sh should be edited so we don't mess with the OS level packages

File: Carrot-Assistant/config.py
Comment (Karthi-DStech) [reply Carrot-Assistant/config.py:1]: Ok James, I'll remove it.

**Review by kuraisle — APPROVED** [review]

**Pull request merged**